### ISSUE TITLE
Server Exception on "wrong" block break

### ISSUE THREAD

**Issue by kreezxil** [einsteinsci/betterbeginnings#77]
Server does not crash, only logs exception. Occurs along with #79.

**Original Post:**

was breaking sandstone block with an iron ingot

```
[17:40:23] [Server thread/FATAL] [FML]: Exception caught executing FutureTask: java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
        at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_60]
        at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_60]
        at net.minecraftforge.fml.common.FMLCommonHandler.callFuture(FMLCommonHandler.java:715) [FMLCommonHandler.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:656) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:364) [po.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:598) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:478) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_60]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
        at net.einsteinsci.betterbeginnings.event.BlockBreakHelper.handleBlockBreaking(BlockBreakHelper.java:102) ~[BlockBreakHelper.class:?]
        at net.einsteinsci.betterbeginnings.event.BBEventHandler.onBlockBreak(BBEventHandler.java:154) ~[BBEventHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_55_BBEventHandler_onBlockBreak_BreakEvent.invoke(.dynamic) ~[?:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:55) ~[ASMEventHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:138) ~[EventBus.class:?]
        at net.minecraftforge.common.ForgeHooks.onBlockBreakEvent(ForgeHooks.java:530) ~[ForgeHooks.class:?]
        at net.minecraft.server.management.ItemInWorldManager.func_180237_b(ItemInWorldManager.java:280) ~[qx.class:?]
        at net.minecraft.server.management.ItemInWorldManager.func_180785_a(ItemInWorldManager.java:241) ~[qx.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:537) ~[rj.class:?]
        at net.minecraft.network.play.client.C07PacketPlayerDigging.func_180763_a(SourceFile:40) ~[ml.class:?]
        at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:10) ~[ml.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[ih.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_60]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_60]
        at net.minecraftforge.fml.common.FMLCommonHandler.callFuture(FMLCommonHandler.java:714) ~[FMLCommonHandler.class:?]
        ... 5 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.resources.I18n
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_60]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_60]
        at net.einsteinsci.betterbeginnings.event.BlockBreakHelper.handleBlockBreaking(BlockBreakHelper.java:102) ~[BlockBreakHelper.class:?]
        at net.einsteinsci.betterbeginnings.event.BBEventHandler.onBlockBreak(BBEventHandler.java:154) ~[BBEventHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_55_BBEventHandler_onBlockBreak_BreakEvent.invoke(.dynamic) ~[?:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:55) ~[ASMEventHandler.class:?]
        at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:138) ~[EventBus.class:?]
        at net.minecraftforge.common.ForgeHooks.onBlockBreakEvent(ForgeHooks.java:530) ~[ForgeHooks.class:?]
        at net.minecraft.server.management.ItemInWorldManager.func_180237_b(ItemInWorldManager.java:280) ~[qx.class:?]
        at net.minecraft.server.management.ItemInWorldManager.func_180785_a(ItemInWorldManager.java:241) ~[qx.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:537) ~[rj.class:?]
        at net.minecraft.network.play.client.C07PacketPlayerDigging.func_180763_a(SourceFile:40) ~[ml.class:?]
        at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:10) ~[ml.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[ih.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_60]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_60]
        at net.minecraftforge.fml.common.FMLCommonHandler.callFuture(FMLCommonHandler.java:714) ~[FMLCommonHandler.class:?]
        ... 5 more
Caused by: java.lang.NullPointerException
```

No crash,

**Comment by einsteinsci** [comment]
I may need to start using packets for this (ugh).

**Comment by einsteinsci** [comment]
This was not meant to be closed...

**Comment by einsteinsci** [comment]
As I18n apparently does not exist on the server, the event stops in the middle and spits out this exception. I reverted the localization support for block breaking from 0.9.3. Fixed in 1.8.

**Comment by einsteinsci** [comment]
Fixed in 1.7.10 too.
